purchase_product_usage/models/purchase_product_usage.py

```diff
@@ -30,15 +30,16 @@ class ProductUsage(models.Model):
         help="Purchase this product by default",
     )
 
-    @api.multi
     def name_get(self):
-        result = super(ProductUsage, self).name_get()
-        new_result = []
-        for usage in result:
-            rec = self.browse(usage[0])
-            if rec.code:
-                name = "[{}] {}".format(rec.code, usage[1])
-            else:
-                name = usage[1]
-            new_result.append((rec.id, name))
-        return new_result
+        for rec in self:
```

Comment:
I don't understand this change, code does not seems ok to me.
@LoisRForgeFlow @AdriaGForgeFlow

Reply:
@HviorForgeFlow at first, I think it's might be multi record but it's seem not necessary. so I'll change to original